packages/process/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Theia process support.",
   "dependencies": {
     "@theia/core": "^0.2.0",
-    "node-pty": "^0.7.0"
+    "node-pty": "^0.7.1"
```

Comment:
why not 0.7.2?

Reply:
They have a broken build. But this did not fix anything. So does not matter.